wdl2cwl/main.py

```diff
@@ -643,6 +656,27 @@ def convert(workflow: str) -> str:
             )
         )
 
+    if "disks" in ast.task_runtime:
+        # get the outdirMin from runtime of disks
+        outdir_min: Union[str, int] = ""
+
+        outdir_min = get_outdir_min(ast.task_runtime["disks"])
+
+        requirements.append(
+            cwl.ResourceRequirement(
+                outdirMin=outdir_min,
+            )
+        )
+    # elif "disks" not in ast.task_runtime:
```

Comment:
Why was this commented out? We should always have an `outdirMin` with the default value if there is no other request from the user. This is required by the OpenWDL 1.1 specifications as I mentioned previously. Yes, you will have to regenerate the other CWL descriptions to match this change.